### DISCUSSION TITLE
Feature request: Autosuggestion dependency type

### DISCUSSION THREAD

**Discussion by indykoning** [wire-elements/spotlight#57] [Ideas]
It would be really nice to have a way to provide autosuggestions to dependencies like the default search type does.
But also have it possible to continue typing and have it act as an input type where you just get the typed text back.

**Comment by PhiloNL** [comment]
Thanks for the feature request @indykoning
Would you mind providing a bit more details with an example use case written in detail so I can get a better understanding on what you are looking for.

**Comment by indykoning** [comment]
In my case i'd like to make a list of named commands to run on a server.
In this case the search dependency would be perfect.
However i also want to be able to run commands that i have typed out fully myself as well. This would require the input dependency.

what i'm thinking of is a search field where the top result is what you typed and is not actually a record in the database.
